[2025-08-08T17:03:40.467-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-08T17:03:40.484-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: avisos_coordenador.consultar_valores_pi manual__2025-08-08T20:03:37.587980+00:00 [queued]>
[2025-08-08T17:03:40.488-0300] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: avisos_coordenador.consultar_valores_pi manual__2025-08-08T20:03:37.587980+00:00 [queued]>
[2025-08-08T17:03:40.488-0300] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-08T17:03:40.591-0300] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): consultar_valores_pi> on 2025-08-08 20:03:37.587980+00:00
[2025-08-08T17:03:40.597-0300] {standard_task_runner.py:63} INFO - Started process 662 to run task
[2025-08-08T17:03:40.599-0300] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'avisos_coordenador', 'consultar_valores_pi', 'manual__2025-08-08T20:03:37.587980+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/avisos_coordenador.py', '--cfg-path', '/tmp/tmpkn9rfri4']
[2025-08-08T17:03:40.600-0300] {standard_task_runner.py:91} INFO - Job 169: Subtask consultar_valores_pi
[2025-08-08T17:03:40.625-0300] {task_command.py:426} INFO - Running <TaskInstance: avisos_coordenador.consultar_valores_pi manual__2025-08-08T20:03:37.587980+00:00 [running]> on host 1a4c58afc681
[2025-08-08T17:03:40.673-0300] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='avisos_coordenador' AIRFLOW_CTX_TASK_ID='consultar_valores_pi' AIRFLOW_CTX_EXECUTION_DATE='2025-08-08T20:03:37.587980+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-08T20:03:37.587980+00:00'
[2025-08-08T17:03:40.674-0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-08T17:03:41.215-0300] {logging_mixin.py:188} INFO - üö® Aviso id_alarme=1 disparado. Sil√™ncio at√© 2025-08-08T20:04:41.215238+00:00
[2025-08-08T17:03:44.545-0300] {logging_mixin.py:188} INFO - üì® Mensagem gerada:
[2025-08-08T17:03:44.546-0300] {logging_mixin.py:188} INFO -  Prezados(as),

Espero que todos estejam bem.

Venho por meio desta trazer √† aten√ß√£o um desvio identificado na gest√£o ambiental de nosso processo.

Conforme ilustrado no gr√°fico de tend√™ncia em anexo, temos observado uma m√©dia de vaz√£o de efluentes gerados pelo processo de produ√ß√£o de papel que supera o limite estipulado em nosso programa de excel√™ncia operacional. Esse limite foi estabelecido em fun√ß√£o da capacidade m√°xima da esta√ß√£o de tratamento de efluentes, que √© exclusiva para a fabrica√ß√£o de papel.

Vaz√£o m√©dia (2 horas): 226,80 m¬≥/h.
[2025-08-08T17:03:49.379-0300] {python.py:237} INFO - Done. Returned value was: None
[2025-08-08T17:03:49.381-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-08T17:03:49.391-0300] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=avisos_coordenador, task_id=consultar_valores_pi, run_id=manual__2025-08-08T20:03:37.587980+00:00, execution_date=20250808T200337, start_date=20250808T200340, end_date=20250808T200349
[2025-08-08T17:03:49.432-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-08T17:03:49.448-0300] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-08T17:03:49.449-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
